extractor/logInterpreter.py

```diff
@@ -46,6 +46,26 @@ def createPacketDict(self, startLineDict, sipHeaderDict, bodyDict):
         return packetDict
 
 
+    def checkForInitialInvite(self, sipHeader, direction, party):
+        sipContent = defaultdict(list)
+        # toWithTag = r'^To:\s*<sip:[A-Za-z0-9+]+@.*>;tag=.*$'
+        toWithoutTag = r'^To:\s*<sip:[A-Za-z0-9+]+@.*>(?!;tag=.*$)'

```

Comment:
Can probably take in sipContent['To'] as toLine
match = re.match(toWithoutTag, toLine)
                    if match and direction == 'from':
                        initialInvite = True
                    print(x)
                    print(initialInvite)
return initialInvite